### DISCUSSION TITLE
appendix/appendix-b-gems

### DISCUSSION THREAD

**Discussion by giscus** [chai2010/advanced-go-programming-book#589] [General]
# appendix/appendix-b-gems

Go语言高级编程

https://chai2010.cn/advanced-go-programming-book/appendix/appendix-b-gems.html

**Comment by ahKevinXy** [comment]
禁止 main 函数退出的方法 有几个是错的吧,只有第一个可以run

**Comment by kanshanlei** [comment]
这个c语言自重写程序要怎么跑起来，main函数没有返回值么？

**Reply by lzwtty** [reply]
main函数默认返回int，函数参数上识别a默认类型char*

**Comment by ZzzJing** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
